### PR TITLE
Feature/docker

### PR DESCRIPTION
**Pull request by Mikazian**
## Description

Please include a summary of the change or which issue is fixed.

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Area of change

- [ ] Frontend
- [ ] Backend
- [x] Other: .....

## General checklist:

- [x] My code follows the style guidelines of this project
- [x] I ran `bun run format`/`bun run lint` before commit
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation where needed
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] I have updated `develop` and merged to my branch before submitting pull request
- [x] My pull request generate no conflicts with `develop` branch
- [x] I requested code review from other team members

## Frontend checklist:

- [ ] I followed guidelines for `HTML`/`JSX`/`Typescript`
- [ ] My `Javascript` generate no new console errors
- [ ] I tested my code cross browsers
- [ ] My slice is pixel perfect for both desktop and mobile according to design
- [ ] I conducted basic QA to assure all features are working
- [ ] I tested responsive for mobile and tablet resolutions

## Backend checklist:

- [ ] I tested admin by manually adding content from zero
- [ ] I created easy to use admin experience which is self-explanatory
- [ ] I added description to admin fields in hard-to-understand areas
- [ ] I followed guidelines for naming `Typescript` variables
- [ ] I conducted basic QA to assure all features are working

### CODE REVIEW

**Review by greegouze — DISMISSED** [review]

File: packages/frontend/web/vite.config.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@ export default defineConfig(({ mode }) => {
       },
     },
     server: {
-      host: FRONTEND_HOST,
+      host: '0.0.0.0',
```
Comment (YDrogen) [inline packages/frontend/web/vite.config.ts:17]: Tu peux supprimer la const au dessus du coup

**Review by YDrogen — CHANGES_REQUESTED** [review]

File: Dockerfile
[the remark this answers is not in the document]
Comment (Mikazian) [reply Dockerfile:1]: supprimé

File: deployment/docker-compose.yml
[the remark this answers is not in the document]
Comment (Mikazian) [reply deployment/docker-compose.yml:1]: supprimé

File: packages/frontend/web/src/components/auth/Login.tsx [hunk 2]
```diff
@@ -61,9 +59,9 @@ export default function Login() {
       // Validation of user entered data using a validation schema
       if (isValid) {
         // Send the login request to the server
-        const res = await fetch(`${API_URL}/auth/login`, {
+        const res = await fetch(`/api/auth/login`, {
           method: 'POST',
-          credentials: 'include', // Send cookies
+          // credentials: 'include', // Send cookies
```
[the remark this answers is not in the document]
Comment (Mikazian) [reply packages/frontend/web/src/components/auth/Login.tsx:64]: on voulait en garder un
j'ai supprimé

File: packages/frontend/web/src/contexts/InteractionContext.tsx [hunk 3]
```diff
@@ -42,7 +42,7 @@ export default function InteractionContext({
     userId,
   });
 
-  // Get the distance between the current user and the selected user
+  // Get the disdockertance between the current user and the selected user
```
[the remark this answers is not in the document]
Comment (Mikazian) [reply packages/frontend/web/src/contexts/InteractionContext.tsx:45]: faute de frappe xD

File: packages/frontend/web/vite.config.ts (hunk 1 above)
Comment (Mikazian) [reply packages/frontend/web/vite.config.ts:17]: retiré

**Review by greegouze — APPROVED** [review]

**Review by YDrogen — APPROVED** [review]

**Pull request merged**